sdk/expo/test/helpers/TestmailInbox.ts

```diff
@@ -0,0 +1,59 @@
+import fetch from 'node-fetch'
```

Comment:
Looks like it's time to move out this file to some shared package instead of copy&pasting it into different projects.

What do you think, @DenisPopov15 ?

Reply:
But also I don't see if this file is referenced anywhere in `expo`?

Reply:
I also don't like copy-pasting, but I'm not sure where to put it. I'd rather have some lib which has useful utilities, including tests -- and would put it there.
And it it referenced in `expo`, here:
https://github.com/affinityproject/affinidi-core-sdk/blob/cba04d447eb3720320f29b89ac39c478dff58d41/sdk/expo/test/integration/otp/index.test.ts#L12

Reply:
We might just create some tiny package, like `@affinidi/testmail-client`

Reply:
Perhaps it can be done as a separate "polishing" task? Because I'm not sure if it's that much of a problem right now (not sure if we gonna change this code any time soon) and I'd rather focus on more important tasks.
@DenisPopov15 what do you think?

Reply:
> And it it referenced in expo, here:

GitHub decided that I don't need to be able to find it 🙃

>  sdk/expo/test/integration/otp/index.test.ts 
> Load diff
> Large diffs are not rendered by default.

Thank you for the tip

Reply:
> Because I'm not sure if it's that much of a problem right now (not sure if we gonna change this code any time soon)

AFAIK we're going to migrate our other projects to use testmail for testing, so it would be really useful to extract it into a separate package.
(And to not have three copies of the same file in affinidi-core-sdk.)

But priorities are up to @DenisPopov15

Reply:
![image](https://user-images.githubusercontent.com/84314823/119528015-84bbc700-bd89-11eb-8659-ad74929c402b.png)

Reply:
> > Because I'm not sure if it's that much of a problem right now (not sure if we gonna change this code any time soon)
> 
> AFAIK we're going to migrate our other projects to use testmail for testing, so it would be really useful to extract it into a separate package.
> (And to not have three copies of the same file in affinidi-core-sdk.)
> 
> But priorities are up to @DenisPopov15

Oh, then it would be useful to extract this, indeed. I need guidance, though, in were to put it and how.
Separate small package (`@affinidi/testmail-client`) that you suggested sounds good to me.